app/inputs/date_time_picker_input.rb

```diff
@@ -36,6 +36,7 @@ def input_html_options(input_name = nil)
     opts[:data].merge!(picker_options: datetime_picker_options)
     opts[:value] ||= input_value(input_name)
     opts[:maxlength] = 19
+    opts[:placeholder] = " "
```

Comment:
Long story short: This is used to show the calendar icon as placeholder.
Explanation:
Our text is longer than activeadmin text (because include the time). So, when the input has text it shows over the image and it's really ugly and hard to read. So, if we want to include the icon, it should work as placeholder (Or maybe other recommendation) It was my first idea.

In this way, we are using the placeholder as behavior. If we wanna do in another way should use a watcher or event (onChange) in javascript related to the input and change the CSS class. For this reason, I rather the placeholder behavior, and take advantage of it.